treatment-service/integration-test/fetch_treatment_it_test.go

```diff
@@ -370,11 +371,11 @@ func (suite *TreatmentServiceTestSuite) SetupSuite() {
 	// Docker compose file copied from official confluentinc repository.
 	// See: https://github.com/confluentinc/cp-all-in-one/blob/7.0.1-post/cp-all-in-one-kraft/docker-compose.yml
 	composeFilePaths := []string{"docker-compose/kafka/docker-compose.yaml"}
-	kafka := testcontainers.NewLocalDockerCompose(composeFilePaths, "kafka")
-	execError := kafka.
-		WithCommand([]string{"up", "-d"}).
-		Invoke()
-	err = execError.Error
+	kafka, err := compose.NewDockerComposeWith(compose.WithStackFiles(composeFilePaths...), compose.StackIdentifier("kafka"))
+	if err != nil {
+		panic(err)
+	}
+	err = kafka.Up(ctx, compose.Wait(true))
```

Comment:
These is the new way we're supposed to run Docker compose (`testcontainers` uses the v2 commands with `compose.NewDockerComposeWith`). I know I mentioned that I took a long time to debug the connectivity issues with the spun up Kafka instance; in the end I realised that the connectivity was working fine all along -.-" I got confused by some logs in the CICD pipeline which made me think that there was an issue with the networking in Docker.